### ISSUE TITLE
Return 0 when get('week')

### ISSUE THREAD

**Issue by cettoana** [cullophid/date-fp#51]
Hi, thanks for this nice lib, it is very helpful for me.

I got some problem when using get('week') as following

``` javascript
const date = new Date(2016, 0, 1);
D.get('week', date); // 0
```

According to the docs and ISO-8601,
Is this result should be 53 instead of 0 ?
Or there are some considerations ?

Many thanks!

**Comment by cullophid** [comment]
Hi @cettoana 
Thank you very much for pointing this out.
this seems to be a bug in our getWeek function https://github.com/cullophid/date-fp/blob/master/src/get.js

I am quite busy at the moment, so It might take a few days before i get to have a look at it.

If you would like to do a PR it would be much appreciated :)
if so please start by adding a failing test :)

**Comment by cettoana** [comment]
No problem, I'll try to work on this issue and do a PR :)
Thanks!

**Comment by cullophid** [comment]
Excellent

**Comment by cullophid** [comment]
ups

**Comment by cullophid** [comment]
Im looking forward to you joining the ranks as a contributor :)

**Comment by cullophid** [comment]
Fixed in version 5.0.2
